app/src/ui/banners/open-thank-you-card.tsx

```diff
@@ -31,8 +31,7 @@ export class OpenThankYouCard extends React.Component<
             emoji={this.props.emoji}
             renderUrlsAsLinks={true}
           />
-          or{' '}
-          <LinkButton onClick={this.onThrowCardAway}>Throw It Away</LinkButton>{' '}
+          or <button onClick={this.onThrowCardAway}>Throw It Away</button>{' '}
```

Comment:
I think we should probably leave these as links unless we are considering changing all the links in the banners to buttons.

Reply:
I missed this one, oops. Fixed now.